libcxx/docs/ReleaseNotes/22.rst

```diff
@@ -38,6 +38,8 @@ What's New in Libc++ 22.0.0?
 Implemented Papers
 ------------------
 
+- P1317R2: Remove return type deduction in ``std::apply`` (`Github <https://github.com/llvm/llvm-project/issues/148183>`__)
```

Comment:
We generally don't include papers that are only partially implemented in the release notes, since that could lead to confusion.

```suggestion
```